edit the docs readme

mirnawong1 opened the pull request:
made additions to readme inspired by [replicated's readme](https://github.com/replicatedhq/replicated-docs). the purpose is to provide more docs explanations and guidance.

there are some open questions for discussion that need to be 🔨 gaveled:

- do we have a naming convention for files? will they diff depending on the type of file (concept, reference, procedural)? Example content below from replicated: 

```
The file name should always start with the feature type (such as dbt-cloud, ide, or job-scheduler). Depending on the content type, it typically also includes a secondary descriptor and a verb. Verbs are used when you are creating a task topic.

Because we author content using Markdown, you must add the `.md` the file extension to the file name.

If you are adding a new page to an existing feature category, follow the existing naming convention for that category.

**Example: Concept file**

`snapshots-backup-hooks.md`

**Example: Reference file**

`snapshots-backup-hooks.md`

**Example: Task file**

`releases-creating-customer.md`

**Example: Tutorial file**

`tutorial-ha-cluster-deploying.md`
```


- what is the difference between alt text and lightbox src for images? Replicated use `![alt text](images/<image-name>.png)` and mention Alt text is used with image syntax, is parsed by screen readers to support accessibility. With *alt text*, the user will hear the alt text spoken out loud by the screen reader/

## Description & motivation
<!---
Describe your changes, and why you're making them. Is this linked to an open
issue, a pull request on dbt core, etc?

To learn more about the writing conventions used in the dbt Labs docs, see the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md).
-->

## To-do before merge
<!---
(Optional -- remove this section if not needed)
Include any notes about things that need to happen before this PR is merged, e.g.:
- [ ] Change the base branch
- [ ] Ensure PR #56 is merged
-->

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

mirnawong1 committed:
augment the docs readme 

made additions to readme that would explain what language is used for docs, how to name files, how to use images, folder structure, callouts, etc. 

there are some open questions for discussion that need to be 🔨 gaveled:

- do we have a naming convention for files? will they diff depending on the type of file (concept, reference, procedural)? Example content below from replicated:
`The file name should always start with the feature type (such as dbt-cloud, ide, or job-scheduler). Depending on the content type, it typically also includes a secondary descriptor and a verb. Verbs are used when you are creating a task topic.

Because we author content using Markdown, you must add the `.md` the file extension to the file name.

If you are adding a new page to an existing feature category, follow the existing naming convention for that category.

**Example: Concept file**

`snapshots-backup-hooks.md`

**Example: Reference file**

`snapshots-backup-hooks.md`

**Example: Task file**

`releases-creating-customer.md`

**Example: Tutorial file**

`tutorial-ha-cluster-deploying.md`
`

- what is the difference between alt text and lightbox src for images? Replicated use `![alt text](images/<image-name>.png)` and mention Alt text is used with image syntax, is parsed by screen readers to support accessibility. With *alt text*, the user will hear the alt text spoken out loud by the screen reader/

mirnawong1 committed:
Merge branch 'current' into mwong-readme-edit

mirnawong1 committed:
Update README.md

mirnawong1 committed:
Update README.md

mirnawong1 committed:
Update README.md

nghi-ly committed:
Merge branch 'current' into mwong-readme-edit

nghi-ly reviewed (commented):
love the content, @mirnawong1 ! i haven't done an editorial review yet but do have a high-level review comment:

the README is starting to get lengthy. should we consider moving these sections (except maybe the "SME and editorial reviews" section) into a guide, maybe even the style guide (so there aren't two guides look in, just one -- one ring to rule them all! )? maybe it can be the "Markdown and style guide" or something?

i've also mentioned this ^ in our slack convo too for more team discussion

runleonarun commented:
Posted in Slack:
> What are the next steps on this?
We could move the new content into the Style guide and rename the content style guide? Ly suggested "Markdown and style guide" or we could call it a "Style and usage guide for [docs.getdbt.com](http://docs.getdbt.com/)" (this would capture the file naming and folder structure content as well as markdown.) (edited

mirnawong1 committed:
Update README.md

migrated changes to style guide. i kept the sme and editorial review section though - wanted to get your thoughts as it may be too much but also thought it may be helpful in the readme.

mshaver631 commented on README.md:
```diff
@@ -23,7 +23,14 @@ You can use components documented in the [docusaurus library](https://v2.docusau
 # Writing content
 
-When writing content, you should refer to the [style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [content types](/contributing/content-types.md) to help you understand our writing standards and how we break down information in the product documentation.
+We author content using Markdown and sometimes HTML. When writing content, you should refer to the [style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [content types](/contributing/content-types.md) to help you understand our writing standards and how we break down information in the product documentation. The dbt Labs Docs team will act ...
```
```suggestion
We author content using Markdown and sometimes HTML. When writing content, refer to the [style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [content types](/contributing/content-types.md) to help you understand our writing standards and how we break down information in the product documentation. The dbt Labs Docs team will actively edit or write content, not just give editorial reviews, and we highly encourage your contributions in the true open-source spirit.
```

mshaver631 replied:
This is great Mirna, thank you for adding more meat to this section.

mirnawong1 committed:
Update README.md

Co-authored-by: mshaver631 <118219367+mshaver631@users.noreply.github.com>

mirnawong1 committed:
Merge branch 'current' into mwong-readme-edit

mirnawong1 committed:
Merge branch 'current' into mwong-readme-edit

mirnawong1 committed:
Merge branch 'current' into mwong-readme-edit

nghi-ly commented on README.md:
```diff
@@ -23,7 +23,14 @@ You can use components documented in the [docusaurus library](https://v2.docusau
 # Writing content
 
-When writing content, you should refer to the [style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [content types](/contributing/content-types.md) to help you understand our writing standards and how we break down information in the product documentation.
+We author content using Markdown and sometimes HTML. When writing content, refer to the [style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [content types](/contributing/content-types.md) to help you understand our writing standards and how we break down information in the product documentation. The dbt Labs Docs team will actively edit  ...
```
```suggestion
The dbt Labs docs are written in Markdown and sometimes HTML. When writing content, refer to the [style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [content types](/contributing/content-types.md) to help you understand our writing standards and how we break down information in the product documentation. 
```

nghi-ly replied:
suggs:

- try to avoid using "we" in techdocs. "we" is ambiguous and can make it harder to translate content: is it the we as in dbt Labs, is it the we as in the reader and the author, or is it we as in everyone? 
- keep content focused on the style guide and contributing process to the dbt Labs docs (vs the Docs team and what the team does)
- "Contributing" section above talks about contributions

nghi-ly commented on README.md:
```diff
@@ -23,7 +23,14 @@ You can use components documented in the [docusaurus library](https://v2.docusau
+
+All PRs that are submitted will be reviewed by the dbt Labs Docs team for editorial review.
+
+Content that is submitted by our users, and the open-source community are also reviewed by our dbt Labs subject matter experts (SMEs) to help ensure technical accuracy.
```
```suggestion
Content that is submitted by our users and the open-source community are also reviewed by our dbt Labs subject matter experts (SMEs) to help ensure technical accuracy.
```

nghi-ly replied:
sugg:

https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#Oxford-comma when there are 3 or more items

nghi-ly reviewed (commented):
tysm for working on the new style and usage guide!

mirnawong1 committed:
Update README.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update README.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Merge branch 'current' into mwong-readme-edit

nghi-ly reviewed (approved):
🚢
